Tests/DependencyInjection/DoctrineCommandsTest.php

```diff
@@ -40,9 +41,12 @@ class DoctrineCommandsTest extends TestCase
      */
     public function testCommandRegistered(string $name, string $instance): void
     {
-        $application = $this->getApplication();
+        $actualCommand = $this->getApplication()->find($name);
```

Comment:
This is a bit confusing… at this point `$actualCommand` might not in fact be the actual command. It will eventually be thanks to line 46 but right now it isn't. I think I would just use `$command` here.

Reply:
Very well.